acme/acme/standalone_test.py

```diff
@@ -266,9 +266,9 @@ def setUp(self):
                     os.path.join(localhost_dir, 'key.pem'))
 
         with closing(socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as sock:
-            sock.bind(('', 0))
-            sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
-            self.port = sock.getsockname()[1]
+            sock.bind(('', 0))  # pylint: disable=no-member
+            sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)  # pylint: disable=no-member
+            self.port = sock.getsockname()[1]  # pylint: disable=no-member
```

Comment:
Can we add `closing` to the list of ignored classes and remove these lines?

Reply:
Done. And for that matter, I removed also all other `no-member`, that are useless now.

Reply:
Thanks for doing that!